lib/live_debugger/app/settings/web/settings_live.ex

```diff
@@ -109,6 +117,7 @@ defmodule LiveDebugger.App.Settings.Web.SettingsLive do
     |> case do
       {:ok, new_settings} ->
         assign(socket, settings: new_settings)
+        push_flash(socket, :info, "Setting updated successfully")
```

Comment:
You need to pipe it to work properly
```suggestion
		socket
        |> assign(settings: new_settings)
        |> push_flash(:info, "Setting updated successfully")
```

Reply:
The funny thing it that I think he does not have to pipe it... 😄  `push_flash/3` does not change anything in the socket - it just  triggers `send(pid, {:put_flash, key, message})` where pid is the parent LiveView. The socket itself is not changed so this could theoretically work. 

But, I'd prefer if this pipe was actually added since we usually assume that this type of functions do change the socket. If we changed this function in the future with that assumption it would break

Reply:
Right, but assigning new settings still should be last then because returned value should be new socket and because of that it's not working properly.